db/seed.js

```diff
@@ -1,4 +1,6 @@
-const connection = require('./connection.js')
+var mysql = require('mysql');
+var config = require('./config');
+var db = mysql.createConnection( config );
 
 // a flexible randomizer that takes arrays or numbers and returns a random element from that range
 var rnd = n => Array.isArray(n) ? n[ rnd(n.length) - 1] : Math.floor( Math.random() * n ) + 1
```

Comment:
Clever function!

Reply:
Thanks! It's fun making little tools like that!